modulus/models/domino/model.py

```diff
@@ -613,16 +608,27 @@ def __init__(
 
         # BQ for surface and volume
         self.neighbors_in_radius = model_parameters.geometry_local.neighbors_in_radius
-        self.radius = model_parameters.geometry_local.radius
-        self.bq_warp = BQWarp(
-            input_features=input_features,
-            grid_resolution=model_parameters.interp_res,
-            radius=self.radius,
-            neighbors_in_radius=self.neighbors_in_radius,
-        )
+        self.radius = model_parameters.geometry_local.radii
+        self.bq_warp = nn.ModuleList()
+        for ct, j in enumerate(self.radius):
```

Comment:
1. What is `ct`?
2. `j` looks like an index, but it's not - it's actual value of a radius.
Maybe refactor it for clarity?